frontend/src/components/Answer/Answer.tsx

```diff
@@ -87,16 +97,18 @@ export const Answer = ({
                             const path = getFilePath(url);
                             return (
                                 <>
-                                    <div style={{
-                                        fontWeight: "500",
-                                        lineHeight: "24px",
-                                        textAlign: "center",
-                                        borderRadius: "4px",
-                                        padding: "0px",
-                                        color: "#123bb6",
-                                        textDecoration: "none"
-                                    }}>
-                                    {`[${++i}]`}
+                                    <div
+                                        style={{
```

Comment:
@ramirezmorac2 shouldn't be this style in a style module instead of inline?

Reply:
That's right, it was already fixed in my last commit